pkg/apis/controller/experiments/v1alpha3/experiment_types.go

```diff
@@ -154,6 +156,13 @@ const (
 	ExperimentFailed     ExperimentConditionType = "Failed"
 )
 
+type ResumePolicyType string
+
+const (
+	NeverResume ResumePolicyType = "Never"
+	LongRunning ResumePolicyType = "LongRunning"
```

Comment:
I like the name `ResumePolicy`.
What do you think about the name `LongRunning`? Is it better than `AlwaysRunning`?
/cc @johnugeorge @gaocegege @richardsliu

Reply:
@gaocegege @johnugeorge @richardsliu  Do you have any ideas about the naming?

Reply:
To make it more consistent, how about keeping options to "Never" and "Always"

Reply:
@johnugeorge  As which proposed in #1062 ,  there will be the third policy type representing "resuming experiment from persistent format"

Reply:
I agree with @sperlingxx.
In case when Experiment is not deleted and always running, `ResumeExperiment: Always` sounds not very obviously.

Reply:
SGTM. I think we can use LongRunning